src/test/java/dev/jbang/cli/TestExternalDeps.java

```diff
@@ -0,0 +1,86 @@
+package dev.jbang.cli;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import dev.jbang.BaseTest;
+import dev.jbang.DecoratedSource;
+import dev.jbang.Util;
+
+import picocli.CommandLine;
+
+class TestExternalDeps extends BaseTest {
+
+	String checkdeps = "public class test {"
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Reply:
Jbang encourages writing scripts that are used from cli. Mixed casing does not work well here. Thus we use lower case for main script entry points. 

I also realize you might just be a bot and don't see this but I do hope you'll take that into your consideration when adding comments elsewhere. 

Peace.